packages/kolibri-common/components/Cards/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+
+  <KCard
+    :to="to"
+    :headingLevel="headingLevel"
+    :layout="windowBreakpoint === 0 ? 'vertical' : 'horizontal'"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    thumbnailScaleType="centerInside"
+    thumbnailAlign="right"
+    :style="{ margin: '16px 0 16px 0', maxWidth: '800px' }"
```

Comment:
Let's remove this before merging too. This was for QA and will be handled by grid.